core/src/main/java/com/klaytn/caver/contract/ContractIOType.java

```diff
@@ -100,4 +115,49 @@ public void setType(String type) {
     public void setIndexed(boolean indexed) {
         this.indexed = indexed;
     }
+
+    /**
+     * Setter function for components
+     * @param components The tuple's components list.
+     */
+    public void setComponents(List<ContractIOType> components) {
+        this.components = components;
+    }
+
+    /**
+     * Returns a type string.
+     * If type string has a tuple, the components of the tuple are listed and returned.
+     * @return String
+     */
+    public String getTypeAsString() {
+        if(getType().contains("tuple")) {
+            String typeString = getComponentAsString();
+            return getType().replace("tuple", "tuple" + typeString);
```

Comment:
```suggestion
            return getType() + typeString;
```
Is this same with this?

Reply:
No, it is not same.
If tuple Array type tuple[], it must represent tuple(typestring)[].

Reply:
Oh, okay :)  Thank you for explaination